### ISSUE TITLE
[@types/react] `placeholder` attribute should not be on HTMLAttributes

### ISSUE THREAD

**Issue by prayerslayer** [DefinitelyTyped/DefinitelyTyped#26736]
- [x] I tried using the `@types/react` package and had problems.
- [ ] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @epzilla @mhegazy @JoshuaKGoldberg

I realize now you may not have seen [my previous comment](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/22933#issuecomment-395091847) because it was on a closed issue.

TLDR: `placeholder` is not a valid attribute on all DOM elements (only input and textarea), therefore #22933 should have been a WONTFIX, and we should revert #24539 because alternative solutions exist, unless [data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) are not an option.

**Comment by pladaria** [comment]
Agree

Something like this:
```ts
interface Props extends React.ImgHTMLAttributes<HTMLImageElement> {
  placeholder?: React.ReactNode;
}
```
should be valid

**Comment by orta** [comment]
Hi thread, we're moving DefinitelyTyped to use [GitHub Discussions](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/53377) for conversations the `@types` modules in DefinitelyTyped.

To help with the transition, we're closing all issues which haven't had activity in the last 6 months, which includes this issue. If you think closing this issue is a mistake, please pop into the [TypeScript Community Discord](https://discord.gg/typescript) and mention the issue in the `definitely-typed` channel.
